libraries/lib-effects/EffectBase.cpp

```diff
@@ -164,7 +164,9 @@ bool EffectBase::DoEffect(EffectSettings &settings,
    std::shared_ptr<EffectInstance> pInstance;
 
    if (IsInteractive()) {
-      if (auto result = finder(settings))
+      if (!finder)
```

Comment:
How could I reach this branch?

Reply:
Build this commit, not the last one, and do the steps to reproduce either issue.  It changes a crash to a quiet failure of the effects.